app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org"
```

Comment:
May I suggest adding a trailing **`/`** to **`HOMEPAGE`**?
I know most browsers hide the trailing slash when there is nothing after it, but, to be correct, a URL needs either a path or a trailing slash.
I also know that this is a detail and that some recipes are missing that trailing slash.
But I think we should have correct URLs in our reicpes.
A practical reason for keeping the slash is that when wrong URLs are copy/pasted to forums that make hyperlinks with them, these hyperlinks do not always work because the wrapper that creates the hyperlinks is not able to correctly detect the end of the domain name when it is followed by a dot. I've already seen that in some places. This never happens with correct URLs ;-)
Well, please feel free to ignore my suggestion if you don't like trailing slashes ;-) Thanks!